jujugui/static/gui/src/app/models/handlers.js

```diff
@@ -88,9 +90,10 @@ YUI.add('juju-delta-handlers', function(Y) {
     */
     createEndpoints: function(endpoints) {
       return Y.Array.map(endpoints, function(endpoint) {
```

Comment:
It would be nice if we kept working towards removing YUI. This can now be done with
```JavaScript 
return endpoints.map(endpoint => {
```